### DISCUSSION TITLE
五笔拼音词库怎么同步

### DISCUSSION THREAD

**Discussion by biange200** [fcitx5-android/fcitx5-android#569] [Q&A | 问答] [answered]
请问一下这个软件怎么同步自己的自定义用户词库

**Comment by rocka** [comment, marked as answer]
输入法的用户数据都保存在 `/sdcard/Android/data/org.fcitx.fcitx5.android/files/` 这个目录里，参考 https://fcitx5-android.github.io/faq/#_8

所有数据都可以通过 高级设置 - 导出用户数据 打包导出。如果你需要多设备同步，可以使用支持 SAF 的同步应用，比如 foldersync.io
